<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preisgruppen</class>
 <widget class="QMainWindow" name="Preisgruppen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1467</width>
    <height>1017</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LÃ¤nder Analyse</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>1161</width>
      <height>971</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>288</width>
      <height>275</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_3"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Personal #: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Figur: </string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="refresh_b">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="b_stats">
       <property name="text">
        <string>Statistiken</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="b_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QFontComboBox" name="fontComboBox"/>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="3" column="1">
      <widget class="QFontComboBox" name="fontComboBox_2"/>
     </item>
     <item row="1" column="1">
      <widget class="QFontComboBox" name="fontComboBox_3"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Arbeitsgang: </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1467</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
